tools/convert_RbacConfig_to_ClusterRbacConfig.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+set -e
+set -u
+
+# This script is provided to help converting RbacConfig to ClusterRbacConfig automatically. The RbacConfig
+# will be deleted after the corresponding ClusterRbacConfig is successfully applied.
```

Comment:
To be precise and to avoid gaps in coverage, the namespace'd RbacConfig should only be deleted after the proxy has adopted new configuration generated by the new cluster scoped RbacConfig. I presume this is what the 15 second delay is intended to address below? If so, that may work most of the time, but still allows for gaps if the apiserver is particularly slow to (or rate limited) to update watches.

Reply:
Yes, you're right for the purpose of the 15 seconds delay.
I'm not aware of any reliable way to check if the new ClusterRbacConfig is applied. Actually this is not only about the API server, but also for Envoy.

Maybe we should not delete in the script and instead let the user to delete it manually?

Reply:
We only need to verify that Pilot has applied the new ClusterRbacConfig. Unfortunately, we can't use status (yet) for this purpose. Something in the logs or metrics may be a useful indicator. The script could tell the user what command to when after manually checking the new cluster-scoped config has been applied.